packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -351,7 +351,7 @@ const MenuItem: VibeComponent<MenuItemProps | MenuItemTitleComponentProps> & {
         {renderMenuItemIconIfNeeded()}
         <Tooltip
           content={shouldShowTooltip ? finalTooltipContent : null}
-          position={tooltipPosition}
+          position={tooltipPosition as TooltipPositionsType}
```

Comment:
I wonder why the ts cast is needed here

Reply:
Thanks, was testing and forgot to delete it